python/lsst/faro/measurement/__init__.py

```diff
@@ -9,3 +9,7 @@
 from .VisitMeasurement import *
 from .TractMeasurement import *
 from .PatchMeasurement import *
+
+# Parquet table versions
```

Comment:
What is this comment for?

Reply:
Currently, we are in the process of transitioning to metric calculations that primarily use parquet table inputs for the source and object catalogs. This is a comment to indicate which of the files are using the parquet table inputs. There is a chance that we might change the directory structure to put the measurements that use parquet table inputs into a separate directory, so my preference is to leave this comment for now.